### DISCUSSION TITLE
How to make the resolution of PDF higher?

### DISCUSSION THREAD

**Discussion by wocoanima** [RyotaUshio/obsidian-pdf-plus#272] [Q&A] [answered]
I don't know why, I just feel a PDF looks much worse in PDF++ of Obsidian than in Adobe/Firefox PDF reader. How do i prevent that? The words are not very clear and sharp. I take a screenshot, you can see that the left hand (edge pdf reader) is much sharper than the right hand side of PDF++ in obsidian. This make my eyes uncomfortable
<img width="1218" alt="image" src="https://github.com/user-attachments/assets/0e1a02d5-630a-47c0-ad08-21cf8c15d625">
![image](https://github.com/user-attachments/assets/25808027-16d9-47be-ab8e-b3e5296c24da)

**Comment by RyotaUshio** [comment, marked as answer]
This is not related to PDF++, it's the core Obsidian issue (unless this problem occurs _only_ when PDF++ is enabled). See #97 and the forum thread therein for some insights.
